client/src/components/Header/__tests__/Header.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Router } from 'react-router';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ThemeProvider from 'components/ThemeProvider';
+import { createMemoryHistory } from 'history';
+import Header from '../Header';
+
+test('renders Inferno Header', () => {
+  render(
+    <ThemeProvider>
+      <Header />
+    </ThemeProvider>
+  );
+
+  const logoElement = screen.getByRole('img');
+  expect(logoElement).toHaveAttribute('alt', 'inferno logo');
+});
+
+test('should navigate home when logo is clicked', () => {
```

Comment:
Not part of this PR, but it would be great if the logo weren't a button, so that you could use it to open the link in a new tab.

Reply:
You mean make the logo a Link or something to open the home page in a new tab? I don't know if that would be expected behavior?

Reply:
Not a new tab as the default behavior, but since it is a button, you can't even cmd/ctrl + click or right click to open in a new tab.

![Screen Shot 2021-12-03 at 1 49 18 PM](https://user-images.githubusercontent.com/15969967/144656750-f5856c56-91a4-4fa7-b73e-5e7179d62744.png)

Reply:
Oh gotcha, I can look into it in the next PR